SOURCES/libpostal-configure.patch

```diff
@@ -49,7 +49,7 @@ index b35f611..58dd670 100644
  
 @@ -11,4 +8,4 @@ TESTS = test_libpostal
  noinst_PROGRAMS = test_libpostal
- test_libpostal_SOURCES = test.c test_expand.c test_parser.c test_transliterate.c test_numex.c test_trie.c test_string_utils.c test_crf_context.c
- test_libpostal_LDADD = ../src/libpostal.la $(CBLAS_LIBS)
+ test_libpostal_SOURCES = test.c test_expand.c test_parser.c test_transliterate.c test_numex.c test_trie.c test_string_utils.c test_crf_context.c ../src/strndup.c ../src/file_utils.c ../src/string_utils.c ../src/utf8proc/utf8proc.c ../src/trie.c ../src/trie_search.c ../src/transliterate.c ../src/numex.c ../src/features.c
+ test_libpostal_LDADD = ../src/libpostal.la ../src/libscanner.la $(CBLAS_LIBS)
 -test_libpostal_CFLAGS = $(CFLAGS_O3)
 +test_libpostal_CFLAGS = $(CFLAGS_BASE)
```

Comment:
Changes to `test/Makefile.am` required a new patch, almost the same as the old one but with minor differences on these two lines.